### DISCUSSION TITLE
Build for releases

### DISCUSSION THREAD

**Discussion by mizaki** [GeoRW/ACBF-Editor#8] [General]
I've yet to look into the problem of outside python dependencies but using `pyinstaller` it should be possible to build for Linux, Windows and MacOS using github actions.

**Comment by GeoRW** [comment]
Pyinstaller looks cool. I have used very complicated process in the past to build it for Windows, then switched to msys2 installer which was easier to use but required like 1 GB of different dependencies until it could build installation package. We can try pyinstaller.
